releases/release-1.23/release-notes/maps/pr-105185-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 105185
+releasenote:
+  text: 'Fix: ignore `not a vmss instance` error for VMAS nodes in `EnsureBackendPoolDeleted`.'
```

Comment:
Ignore `not a vmss instance` error for VMAS nodes in `EnsureBackendPoolDeleted`.

Reply:
I guess ignore does not need to be camel since it is not the start word^

Reply:
Like I saw it in the release notes draft PR for Alpha.2 that the 'Fix' word was replaced and the sentence was rephrased accordingly.

Reply:
`Fix:` is more of an style preference as long as the statement is clear. Since the statement itself is pretty clear, I am ok with either. :)